catalog-info.yaml

```diff
@@ -0,0 +1,46 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+# (Required) Acceptable Values: Component, Resource, System
+# Use `Component` unless you know how backstage works and what the other kinds mean.
+kind: Component
+metadata:
+  # (Required) Must be the name of the repo, without the owning organization.
+  name: 'user-util'
+  description: "User utilities for the Open edX platform"
+  links:
+    - url: "https://docs.openedx.org"
+      title: "User Util"
+      # Backstage uses the MaterialUI Icon Set.
+      # https://mui.com/material-ui/material-icons/
+      # The value will be the name of the icon.
+      icon: "Web"
+  annotations:
+    # (Optional) Annotation keys and values can be whatever you want.
+    # We use it in Open edX repos to have a comma-separated list of GitHub user
+    # names that might be interested in changes to the architecture of this
+    # component.
+    openedx.org/arch-interest-groups: "feanil"
+    # (Optional) We use the below annotation to indicate whether or not this
+    # repository should be tagged for openedx releases and which branch is tagged.
+    openedx.org/release: "main"
+spec:
+
+  # (Required) This can be a group (`group:<github_group_name>`) or a user (`user:<github_username>`).
+  # Don't forget the "user:" or "group:" prefix. Groups must be GitHub team
+  # names in the openedx GitHub organization: https://github.com/orgs/openedx/teams
+  #
+  # If you need a new team created, create an issue with Axim engineering:
+  # https://github.com/openedx/axim-engineering/issues/new/choose
+  owner: user:ichintanjoshi
```

Comment:
@ichintanjoshi unfortunately we can't make you an owner until after you're a CC, I think for now, you should remove this file and we'll come back to it at a later time.

Reply:
Okay, no worries, I'll remove it, just added it because of section in docs.